cbt_enable_and_snapshot_remote.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import sys
+import XenAPI
+import shutil
+import urllib3
+import requests
+from xmlrpclib import ServerProxy
+
+def export_vdi(host, session_id, vdi_uuid, file_format, export_path):
+    url = ('https://%s/export_raw_vdi?session_id=%s&vdi=%s&format=%s'
+           % (host, session_id, vdi_uuid, file_format))
+    with requests.Session() as session:
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        request = session.get(url, verify=False, stream=True)
+        with open(export_path, 'wb') as filehandle:
+            shutil.copyfileobj(request.raw, filehandle)
+        request.raise_for_status()
+
+# Add your host address here
+host = "cl14-02.xenrt.citrite.net"
+vdi_uuid = sys.argv[1]
+
+p = ServerProxy("http://" + host)

```

Comment:
Please use the child class directly, e.g. https://github.com/xapi-project/xen-api/blob/master/scripts/examples/python/XenAPI.py#L122

Reply:
Please also always use https.

Reply:
when connecting using https I see the error "ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)". I assume that the xapi-ssl.pem certificate will also need to be passed in the connection? (Will test later after fixing up other comments)